src/performance/gpu/amd/asus.rs

```diff
@@ -0,0 +1,95 @@
+use std::sync::Arc;
```

Comment:
Is this file necessary for this PR? It doesn't appear to be used yet and most of the branches are incomplete anyway. It also seems to be out of scope for the PR subject.

Reply:
This file exists since the previous PR (as a placeholder, actually unused) and so I had to edit it to met the new interface.

I tried to minimize modification on this file, but that line could have sneaked in.

I will see if removing such line makes the build success.

Reply:
Oh, I guess I missed that in the last PR. I suppose its fine then since you're just aligning everything to async and the calls to this file are commented out. In the future please try to be a good citizen of git and keep all changes related to the goal a specific PR, it makes reverting breaking changes and managing merge conflicts easier in the future.

Reply:
I am sorry, those sneaked in in the previous PR. I had them initially to test that the mechanism of specializing the hardware was working and forgot to remove them. I am still in time to remove all of those just to re-add them later.